### PR TITLE
Make the Tokenizer Classy

### PR DESCRIPTION
**Pull request by jonthegeek**
Checking this in so you can go from here. Mostly all I did was delete stuff, but I started doing a little bit of organization toward the "include the tokens in the output and also add a class" ticket.

### CODE REVIEW

**Comment by jonathanbratt** [comment]
@jonthegeek  Not sure the best way to do this. This branch is now ready for your review, but you're the owner of the draft PR. You can just review here, if you want, and I will "review" when you're done.

File: R/tokenize.R [hunk 1]
```diff
@@ -476,13 +454,15 @@ tokenize_bert <- function(...,
...
+    increment_index = increment_index
+  )
   return(
-    .finalize_bert_tokens(
-      tokenized_text = token_ids,
-      token_types = token_type_ids,
-      increment_index = increment_index,
-      simplify = simplify
-    )
+    structure(to_return,
```
Comment (jonthegeek) [inline R/tokenize.R:463]: I feel like this should be inside `.finalize_bert_tokens` (thus the name), and then directly return the result of that call. Is there a reason you put it here?

**Review by jonthegeek — COMMENTED** [review]
I can't formally request changes since it's mine but pretend that I requested a change (or at least a discussion)! Just that one!

File: R/tokenize.R (hunk 1 above)
Comment (jonathanbratt) [reply R/tokenize.R:463]: I thought about putting it there, but this... felt better? I don't have a good argument for why, though. :)

Comment (jonathanbratt) [reply R/tokenize.R:463]: ...but I think you're right, because sometimes we return earlier than this! ok, I'll change it.

Comment (jonathanbratt) [reply R/tokenize.R]: This should be fixed now. Back to you!

**Review by jonthegeek — COMMENTED** [review]
Looks good! Now it's your turn to review!

**Review by jonathanbratt — APPROVED** [review]

**Pull request merged**